mod/blog/actions/blog/auto_save_revision.php

```diff
@@ -1,10 +1,10 @@
 <?php
+
 /**
  * Action called by AJAX periodic auto saving when editing.
  *
  * @package Blog
  */
-
 $guid = get_input('guid');
 $user = elgg_get_logged_in_user_entity();
 $title = htmlspecialchars(get_input('title', '', false), ENT_QUOTES, 'UTF-8');
```

Comment:
I think the idea is to not strip tags but to encode them, e.g. you can have a blog titled ``How to use <a> tags``